### ISSUE TITLE
pkgdown building failure

### ISSUE THREAD

**Issue by maelle** [ropensci/rcites#68]
:wave: @KevCaz!

We see an error on https://github.com/r-universe/ropensci/actions/runs/3785494594/jobs/6551249470 via https://ropensci.r-universe.dev/ui#builds

Is it possible that examples require authentication? (Running them locally I get `Unauthorized (HTTP 401)`) If so could you please wrap them in `\dontrun{}`?

**Comment by KevCaz** [comment]
Looks like there is still an encoding issue on Windows (oldrel) 

```
 Last 13 lines of output:
  == Failed tests ================================================================
  -- Failure ('test-distributions.R:44'): spp_distributions() language works -----
  lang_GQ %in% res1$distributions$name (`actual`) not identical to c(FALSE, TRUE, FALSE) (`expected`).
  
  `actual`:   FALSE FALSE FALSE
  `expected`: FALSE TRUE  FALSE
  -- Failure ('test-legislations.R:47'): spp_cites_legislation() scope & language works --
  "Guin�e" %in% res$cites_suspensions$geo_entity.name is not TRUE
```

**Comment by KevCaz** [comment]
```
Last 13 lines of output:
  == Failed tests ================================================================
  -- Failure ('test-distributions.R:44'): spp_distributions() language works -----
  lang_GQ %in% res1$distributions$name (`actual`) not identical to c(FALSE, TRUE, FALSE) (`expected`).
  
  `actual`:   FALSE FALSE FALSE
  `expected`: FALSE TRUE  FALSE
  -- Failure ('test-legislations.R:47'): spp_cites_legislation() scope & language works --
  "Guin�e" %in% res$cites_suspensions$geo_entity.name is not TRUE
  
  `actual`:   FALSE
  `expected`: TRUE 
  
  [ FAIL 2 | WARN 0 | SKIP 6 | PASS 105 ]
  Error: Test failures
```

I guess one way to deal with that is to use a transliteration in the test. The problem is that I cannot reproduce the issue locally and neither in the oldrel on windows for my branch...

**Comment by maelle** [comment]
I have now registered for an API token! :grin: I don't use Windows though.

> I guess one way to deal with that is to use a transliteration in the test. 

What do you mean? Using another country name without accent?

> The problem is that I cannot reproduce the issue locally and neither in the oldrel on windows for my branch...

Are you on Windows? It'd be interesting to know what the geo_entity.name looks like on Windows where it fails. A way to do that would be to use a snapshot test, this way in the check artefacts you'd see how the snapshot looks like on Windows. Then maybe it'd help see where to add something related to encoding?

Another idea: does the problem appear when not using a cassette?

**Comment by KevCaz** [comment]
@maelle Sorry I missed your message 😞 ... Looks like everything is ok now!
 
> What do you mean? Using another country name without accent?
What I ended up doing is escape the non-ascii characters. Not sure it worked, though since I think the oldrel version has changed since I did the changes. 

> Are you on Windows? 

No I am not.  

> to use a snapshot test, 

I will definitively use snapshot in my test in the next release! 

If it's ok with you, I will close the issue.
